dsc/workflows/base/__init__.py

```diff
@@ -221,6 +222,58 @@ def item_metadata_iter(self) -> Iterator[dict[str, Any]]:
         MUST be overridden by workflow subclasses.
         """
 
+    def create_batch(self) -> None:
+        """Create a batch of item submissions for processing.
+
+        A "batch" refers to a collection of item submissions that are grouped together
+        for coordinated processing, storage, and workflow execution. Each batch
+        typically consists of multiple items, each with its own metadata and
+        associated files, organized under a unique batch identifier.
+
+        This method prepares the necessary assets in S3 (programmatically as needed)
+        and records each item in the batch to DynamoDB.
+        """
+        item_submissions, errors = self.prepare_batch()
+        if errors:
+            raise BatchCreationFailedError
```

Comment:
Is this a bit of a stubbed exception handling?  FWIW, totally fine with that.  I ask, because it feels like we're not doing anything with the `errors` output which could be helpful for reporting.